File: LocalazyCard/src/main/java/com/paget96/localazycard/utils/UiUtils.java
package com.paget96.localazycard.utils;

import android.content.ActivityNotFoundException;
import android.content.Context;
import android.content.Intent;
import android.net.Uri;
import android.util.AndroidRuntimeException;
import android.view.View;

public class UiUtils {

    // Parses and open links.
    public static void openLink(Context context, String link) {
        Uri uri = Uri.parse(link);
        Intent intent = new Intent(Intent.ACTION_VIEW, uri);
        try {
            context.startActivity(intent);
        } catch (ActivityNotFoundException e) {
            e.printStackTrace();
        }
    }
}

File: LocalazyCard/src/main/java/com/paget96/localazycard/utils/UiUtils.kt
package com.paget96.localazycard.utils

import android.content.ActivityNotFoundException
import android.content.Context
import android.content.Intent
import android.net.Uri

object UiUtils {
    // Parses and open links.
    @JvmStatic
    fun openLink(context: Context, link: String?) {
        val uri = Uri.parse(link)
        val intent = Intent(Intent.ACTION_VIEW, uri)
        try {
            context.startActivity(intent)
        } catch (e: ActivityNotFoundException) {
            e.printStackTrace()
        }
    }
}